internal/cmd/schema-registry/command_schema_delete.go

```diff
@@ -67,24 +67,33 @@ func deleteSchema(cmd *cobra.Command, srClient *srsdk.APIClient, ctx context.Con
 		return err
 	}
 
+	permanent, err := cmd.Flags().GetBool("permanent")
+	if err != nil {
+		return err
+	}
+
 	checkVersion := version
 	if version == "all" {
 		// check that at least one version for the input subject exists
 		checkVersion = "latest"
 	}
-	_, httpResp, err := srClient.DefaultApi.GetSchemaByVersion(ctx, subject, checkVersion, nil)
-	if err != nil {
+	if permanent {
+		getOpts := &srsdk.GetSchemaByVersionOpts{Deleted: optional.NewBool(true)}
```

Comment:
```suggestion
		opts := &srsdk.GetSchemaByVersionOpts{Deleted: optional.NewBool(true)}
```

Reply:
Thanks! This was an artifact from an earlier iteration where I didn't want it conflict with `opts` defined further down and I forgot to switch it back after I moved it inside an if block.